docs/01-overview/main-areas/application-connectivity/README.md

```diff
@@ -5,11 +5,15 @@ title: What is Application Connectivity in Kyma?
 Application Connectivity in Kyma is an area that: 
 
 - Simplifies and secures the connection between external systems and Kyma
+<!-- TODO: is this one still true?
 - Registers external events and APIs and simplifies the API usage
```

Comment:
I would say it stores and handles metadata, we are not registering it anymore but we still have it in Application CR